packages/util/src/obj.ts

```diff
@@ -18,12 +18,20 @@
 /* eslint-disable */
 // will enable after https://github.com/firebase/firebase-js-sdk/pull/1811 is merged
 
+interface UtilObject<V> {
+  [key: string]: V;
+  [index: number]: V;
```

Comment:
No need to define a number index. All number indexing are string indexing under the hood.

Reply:
Weird, I was previously getting a lot of errors in places that used this functions with number-indexed objects but I guess those errors got fixed on the consuming end.  Done.